<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="table"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnScan">
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="log"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
